src/commands/compareReports.js

```diff
@@ -56,33 +56,40 @@ export default async function compareReports(
   if (dryRun) {
     log('Running in --dry-run mode -- no destructive commands will be issued');
   }
-  await Promise.all(
-    firstCompareResult.diffs.map(async ([before, after]) => {
-      const firstDiffDistance = await compareSnapshots({
-        before,
-        after,
-        endpoint,
-        compareThreshold,
-      });
-      if (!firstDiffDistance) {
-        log(
-          `✓ ${after.component} - ${after.variant} - ${
-            after.target
-          } - diff is within threshold`,
-        );
-        if (!dryRun) {
-          await ignore({ before, after, apiKey, apiSecret, endpoint });
+
+  const diffsClone = firstCompareResult.diffs.slice(0);
+  let batch;
+  // eslint-disable-next-line no-cond-assign
+  while ((batch = diffsClone.splice(0, 10)).length > 0) {
+    // eslint-disable-next-line no-await-in-loop
+    await Promise.all(
+      batch.map(async ([before, after]) => {
+        const firstDiffDistance = await compareSnapshots({
+          before,
+          after,
+          endpoint,
+          compareThreshold,
+        });
+        if (!firstDiffDistance) {
+          log(
+            `✓ ${after.component} - ${after.variant} - ${
+              after.target
+            } - diff is within threshold`,
```

Comment:
Unrelated, I think this would be a clearer thing to log

```suggestion
            } - diff is below threshold, auto-ignoring`,
```